packages/node/src/machine/types.ts

```diff
@@ -104,6 +106,9 @@ export type InstallVirtualAppParams = {
   initiatorBalanceDecrement: BigNumber;
   responderBalanceDecrement: BigNumber;
   tokenAddress: string;
+
+  // outcomeType returned by the app instance, as defined by the app definition `appInterface`
```

Comment:
Is this comment necessary? None of the other fields have comments.

Reply:
The other fields really should have comments